primehub/datasets.py

```diff
@@ -385,3 +396,15 @@ def validate_update(config):
     }
     """)
     spec.validate(config)
+
+
+def validate_id(id: str):
```

Comment:
Convert this to a new type for ValidationSpec: `ResourceId` or another better name

Reply:
it shouldn't duplicate, just call the function from validation module

Reply:
ok, will integrate with `validate_name` fuction in the validation module